README.md

```diff
@@ -1,13 +1,10 @@
-[![Bors enabled](https://bors.tech/images/badge_small.svg)](https://app.bors.tech/repositories/61986)
-
 # pavel.yaiwr
 
-Yet Another Interpreter Written In Rust
+Yet Another Interpreter Witten In Rust
```

Comment:
Are you sure the rebase was correct? Didn't this change appear in a prior PR?

Reply:
that's weird, will investigate :detective: